Tasks/UseRubyVersionV0/Tests/L0.ts

```diff
@@ -1,163 +1,76 @@
-import * as assert from 'assert';
-import { EOL } from 'os';
-import * as path from 'path';
-
-import * as mockery from 'mockery';
-import * as mockTask from 'vsts-task-lib/mock-task';
-import * as useRubyVersion from '../userubyversion';
-
-/** Reload the unit under test to use mocks that have been registered. */
-function reload(): typeof useRubyVersion {
-    return require('../userubyversion');
-}
+import fs = require('fs');
+import assert = require('assert');
+import path = require('path');
```

Comment:
The `import ... require` syntax is made for compat with some other JS module loaders, but the ES6-standard `import * as path from 'path';` gets treated more strictly and is usually what you want.

For example, TypeScript will treat all of the properties on the module imported with `import as` as read-only, where with `require` it will let you reassign them.

Reply:
Will update, we have both styles in the same file which is not great. Is there a order of imports convention in TS?

Reply:
From looking at some other tasks it looks like we do Node modules first, then task lib, then 3rd-party modules.